README.md

```diff
@@ -7,6 +7,59 @@ Node-API contract from Node.js allows consumers of this library to interact with
 JavaScript engine with a consistent interface. This library also provides some optional
 polyfills that consumers can include if required.
 
+
+## **Building - All Development Platforms**
+
+**Required Tools:** [git](https://git-scm.com/), [CMake](https://cmake.org/), [node.js](https://nodejs.org/en/)
+
+The first step for all development environments and targets is to clone this repository. 
+
+Use a git-enabled terminal to follow the steps below. The `--recursive` flag is necessary as 
+Babylon Native makes extensive use of submodules for dependencies.
+
+```
+git clone --recursive https://github.com/BabylonJS/JsRuntimeHost.git
+```
+
+Babylon Native requires Babylon.js. You will need to install NPM packages to resolve these dependencies. From the root of the `JsRuntimeHost` 
+repository on the command line, run the following commands:
```

Comment:
I hope to remove this step some day. But in the meantime, this text is still off. :)

```suggestion
The unit tests require some NPM packages. From the root of the repository on the command line, run the following commands:
```